VMDIRAC/Resources/Cloud/CloudEndpoint.py

```diff
@@ -500,6 +501,8 @@ def assignFloatingIP( self, node ):
       pool = result['Value']
       try:
         floatingIP = pool.create_floating_ip()
+        ## Add sleep between creation and assignment
+        time.sleep( 60 )
```

Comment:
At least for one cloud provider I've realized that without this delay the ressource was considered busy and the IP assignment systematically failed.